pkg/operator/ceph/pool/validate.go

```diff
@@ -33,6 +34,11 @@ func validatePool(context *clusterd.Context, clusterInfo *client.ClusterInfo, cl
 	if p.Namespace == "" {
 		return errors.New("missing namespace")
 	}
+
+	if err := v1.ValidateCephBlockPool(&p.Spec); err != nil {
+		return err
+	}
+
```

Comment:
Now the CephBlockPool reconciler will use the same code as the admission controller for some of its validation.